docker-compose-production.yml

```diff
@@ -0,0 +1,40 @@
+app:
+  build: .
+web:
+  extends:
+    service: app
+  command: gunicorn -b 0.0.0.0:8000 --reload measurements.wsgi --workers 4 --timeout 120
+  ports:
+    - "8000"
+  links:
+    - redis
+    - rabbitmq
+https-portal:
+  image: steveltn/https-portal:1.0.0
+  ports:
+    - "80:80"
+    - "443:443"
+  links:
+    - web
```

Comment:
Apart from this there are a number of things introduced in `docker-compose-production.yml`:
- Removal of the db container
- Addition of the https-portal

@hellais Do you think that we can get these changes in `config/production.yml` ?